packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -88,10 +106,29 @@ const ActionHeader = ({
   onPrevious,
   onNext,
   children,
+  backButtonA11yLabel,
+  nextButtonA11yLabel,
+  prevButtonA11yLabel,
   ...customProps
 }) => {
   const theme = React.useContext(ThemeContext);
 
+  const descriptiveText1 = `On Clicking the close button, ${text} will be closed`;
+  const descriptiveText2 = `On Clicking the maximize button, ${text} will be maximized`;
+  const descriptiveText3 = `On Clicking the minimize button, ${text} will be minimized`;
```

Comment:
Should these text be translated?

Reply:
Yes, I have asked for Neil's approval for the text and will log the translation JIRA and replace these.